lib/renderer.js

```diff
@@ -91,6 +92,13 @@ Renderer.prototype.paragraph = function(text) {
 marked.setOptions({
   langPrefix: '',
   highlight: function(code, lang) {
+    lang = String(lang).toLowerCase();
+    if (lang === 'math' || lang === 'katex' || lang === 'latex') {
+      return katex.renderToString(code, {
```

Comment:
I wonder if this can be integrated into [`highlight()`](https://github.com/hexojs/hexo-util/#highlightstr-options)?